conf/gconf/cluster.conf.sample

```diff
@@ -55,7 +55,7 @@ cortx:
     agent:
       num_endpoints: 1
       endpoints:
-      - https://:8081
+      - https://:23256
```

Comment:
Please Update this port in confluence pages also to keep these sample files in sync with confluence pages
@ujjwalpl  @nitin-seagate

Reply:
I do not have edit access to the page.